src/nohash.rs

```diff
@@ -0,0 +1,23 @@
+use std::hash::{BuildHasherDefault, Hasher};
+
+/// Inspired by nohash-hasher, but we avoid the crate dependency because it's in public archive.
+#[derive(Copy, Clone, Debug, Default)]
+pub struct NoHashHasher(u64);
+
+pub type BuildNoHashHasher = BuildHasherDefault<NoHashHasher>;
```

Comment:
I think these two can be more properly restricted to `pub(crate)`, right?

Reply:
Yeah. They're technically never re-exported, but I've adjusted to `pub(crate)` per your request so that it's explicit to the reader.